test/staking/ClusterRewards.ts

```diff
@@ -912,7 +912,7 @@ describe("ClusterRewards submit tickets", function() {
     await time.increaseTo(startTime + 34 * 86400);
 
     // reverted because of overflow in args
-    await expect(clusterRewards.connect(signers[5])["issueTickets(bytes32,uint24,uint16[])"](ETHHASH, epochWithRewards, [MAX_TICKETS_1_pc.mul(10), MAX_TICKETS_1_pc.mul(20), MAX_TICKETS_1_pc.mul(30), MAX_TICKETS.add(1)])).to.be.revertedWith("");
+    await expect(clusterRewards.connect(signers[5])["issueTickets(bytes32,uint24,uint16[])"](ETHHASH, epochWithRewards, [MAX_TICKETS_1_pc.mul(10), MAX_TICKETS_1_pc.mul(20), MAX_TICKETS_1_pc.mul(30), MAX_TICKETS_1_pc.mul(40).add(10000)])).to.be.revertedWith("CRW:IPRT-Total ticket count invalid");
```

Comment:
is 10000 the minimum I need to add to get it to overflow?

Reply:
no, 
it was added because some values got floored due to division. 
Changed it now